### PR TITLE
Add web config localization lables

### PR DESCRIPTION
**Pull request by rebornsick**
I added these localization lables by imitating those previous PRs. Compiled and tested by myself, looks good to me.

### CODE REVIEW

File: www/src/Locales/ja-JP/InputMacroAddon.jsx [hunk 1]
```diff
@@ -11,12 +11,6 @@ export default {
 		'起動にマクロボタン＋コントローラボタンを利用',
 	'input-macro-macro-button-pin-plus': 'ボタン割り当て：',
 	'input-macro-header-text': 'マクロ設定概要',
-	'input-macro-1-header-text': 'マクロ 1 設定',
-	'input-macro-2-header-text': 'マクロ 2 設定',
-	'input-macro-3-header-text': 'マクロ 3 設定',
-	'input-macro-4-header-text': 'マクロ 4 設定',
-	'input-macro-5-header-text': 'マクロ 5 設定',
-	'input-macro-6-header-text': 'マクロ 6 設定',
```
Comment (mizma) [inline www/src/Locales/ja-JP/InputMacroAddon.jsx:19]: if you could add `'input-macro-macro-list-txt': 'マクロ {{macroNumber}} 設定'` there that would be nice.

Comment (rebornsick) [reply www/src/Locales/ja-JP/InputMacroAddon.jsx:19]: sure, anything else?

Comment (mizma) [reply www/src/Locales/ja-JP/InputMacroAddon.jsx:19]: no I'll work on the other new labels once this is merged.

**Review by arntsonl — APPROVED** [review]
Looks great, thank you!

**Pull request merged**